cypress/component/Forms/SelectOptions.spec.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import {mount} from 'cypress/react'
+import {FormField, FormFieldTypes} from '../../../src/components/forms/forms';
+import {asIdAndDisplayText, getFormattedName} from '../../../src/components/forms/SelectOptionInterface';
+import {StudyType} from '../../../src/components/forms/StudyType';
+import {SecondaryDataUseTerms} from '../../../src/components/forms/SecondaryDataUseTerms';
+import {NIHInstitutesAndCenters} from '../../../src/components/forms/NIHInstitutesAndCenters';
```

Comment:
Is this a test of `FormField` or of `StudyType`, `SecondaryDataUseTerms`, `NIHInstitutesAndCenters`?

Unless the data objects are being tested, I think it's better to use data defined in the test code only, to isolate the test from the test data.

If the data objects do require a test, it would be better to test those in isolation as well